e2e-tests/src/main/java/uk/nhs/adaptors/gp2gp/MessageQueue.java

```diff
@@ -0,0 +1,41 @@
+package uk.nhs.adaptors.gp2gp;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionFactory;
+import javax.jms.Destination;
+import javax.jms.MessageProducer;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import java.util.Hashtable;
+
+public class MessageQueue {
+
+    public static void sendToMhsInboundQueue(String messageContent) throws Exception {
+        String broker = System.getenv().getOrDefault("GP2GP_AMQP_BROKERS", "amqp://localhost:5672");
+        String queueName = System.getenv().getOrDefault("GP2GP_MHS_INBOUND_QUEUE", "inbound");
+
+        Hashtable<String, String> hashtable = new Hashtable<>();
```

Comment:
 Consider replacing this Hashtable with the newer java.util.Map

Reply:
InitialContext requires a Hashtable